modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-private-image-registry-private-azure"]
+= Optional: Preparing a private Microsoft Azure cluster for a private image registry
+
+By installing a private image registry on a private Microsoft Azure cluster, you can create private endpoints. Private endpoints disable public facing endpoints to the internal registry, adding an extra layer of security to your {product-title} deployment.
+
+Use the following guide to prepare your private Microsoft Azure cluster for installation with a private image registry.
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have prepared an `install-config.yaml` that includes the following information:
+** A file that includes your VPC configuration
+** The `publish` field is set to `Internal`
+** The `credentialsMode` field is set to `Manual`
+
+* You have created a global Microsoft Azure account for the `ccoctl` utility.
+
+.Procedure
+
+. If you have not previously created installation manifest files, do so by running the following command:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir <installation_directory>
+----
++
+This command displays the following messages:
++
+.Example output
+[source,terminal]
+----
+INFO Consuming Install Config from target directory
+INFO Manifests created in: <installation_directory>/manifests and <installation_directory>/openshift
+----
+
+. Create an image registry configuration object and pass in the `networkResourceGroupName`, `subnetName`, and `vnetName` provided by Microsoft Azure. For example:
++
+[source,terminal]
+----
+$ touch imageregistry-config.yaml
+----
++
+[source,yaml]
+----
+apiVersion: imageregistry.operator.openshift.io/v1
+kind: Config
+metadata:
+  name: cluster
+spec:
+  managementState: "Managed"
+  replicas: 2
+  rolloutStrategy: RollingUpdate
+  storage:
+    azure:
+      networkAccess:
+        internal:
+          networkResourceGroupName: <vnet_resource_group> <1>
+          subnetName: <subnet_name> <2>
+          vnetName: <vnet_name> <3>
+        type: Internal 
+----
+<1> Replace `<vnet_resource_group>` with the resource group name that contains the existing virtual network (VNet).
+<2> Replace `<subnet_name>` with the name of the existing subnet within the specified resource group.
+<3> Replace `<vnet_name>` with the name of the existing virtual network (VNet) in the specified resource group.
+
+. Move the `imageregistry-config.yaml` file to the `<installation_directory/manifests>` folder by running the following command:
```

Comment:
don't know if it matters here but this file is is consumed after installation.

Reply:
Add a note:

[NOTE]
====
The `imageregistry-config.yaml` file is consumed during the installation process. If desired, you must back it up before installation.
====